code/iaas/model/src/main/java/io/cattle/platform/core/cleanup/TableCleanUp.java

```diff
@@ -37,6 +39,19 @@ public void cleanUp(){
                 .execute();
 
         long endProcessInstance = System.currentTimeMillis();
+
+        long startProcessExecution = System.currentTimeMillis();
+        long lowestProcessInstance = create()
```

Comment:
@hibooboo2 process_executions are already deleted as part of a cascading deleting from process_instance.  So the PR effectively does nothing new.  I think what we want is just to delete any process_execution record that is older than a day, just like we delete process_instance.

Because of the cascading delete we should delete process_instance first and then look for and delete process_execution.